src/app/components/Timer.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component, PropTypes } from 'react';
+
+export default class Timer extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            secondsElapsed: props.from
+        };
+    }
+
+    tick() {
+        this.setState({
+            secondsElapsed: this.state.secondsElapsed + 1
+        });
+    }
+
+    componentDidMount() {
+        this.interval = setInterval(() => this.tick(), 1000);
+    }
+
+    componentWillUnmount() {
+        clearInterval(this.interval);
+    }
+
+    render() {
+        return (
+            <div>Seconds Elapsed: {this.state.secondsElapsed}</div>
+        );
+    }
+}
+
+Timer.propTypes = {
```

Comment:
Class properties are still experimental in [stage-2](http://babeljs.io/docs/plugins/preset-stage-2/). So here should come discussion which preset we should use. I would choose the [latest](http://babeljs.io/docs/plugins/preset-latest/) preset.